exercises/list-ops/README.md

```diff
@@ -0,0 +1,54 @@
+# List Ops
+
+Implement basic list operations.
+
+In functional languages list operations like `length`, `map`, and
+`reduce` are very common. Implement a series of basic list operations,
+without using existing functions.
+
+The precise number and names of the operations to be implemented will be 
+track dependent to avoid conflicts with existing names, but the general
+operations you will implement include:
+
+* `append` (*given two lists, add all items in the second list to the end of the first list*);
+* `concatenate` (*given a series of lists, combine all items in all lists into one flattened list*);
```

Comment:
This is not actually checked in this implementation. This is because as C does not allow dynamic typing, the list implementation can only be for one type at a time. Concatenation implies that the list implementation is used for data type, and list type in parallel.